File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/Rating.java
package com.drivingschoolrwandaapp.models.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Rating implements Serializable {
    @SerializedName("id")
    private int id;

    @SerializedName("pdfId")
    private int pdfId;

    @SerializedName("rating")
    private double rating;

    @SerializedName("comment")
    private String comment;

    @SerializedName("user")
    private User user;

    @SerializedName("createdAt")
    private String createdAt;

    // Getters and Setters
    public int getId() { return id; }
    public int getPdfId() { return pdfId; }
    public double getRating() { return rating; }
    public String getComment() { return comment; }
    public User getUser() { return user; }
    public String getCreatedAt() { return createdAt; }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/Rating.kt
package com.drivingschoolrwandaapp.models.entities

import com.google.gson.annotations.SerializedName

data class Rating(
    @SerializedName("id") var id: Int = 0,
    @SerializedName("pdfId") var pdfId: Int = 0,
    @SerializedName("rating") var rating: Double = 0.0,
    @SerializedName("comment") var comment: String? = null,
    @SerializedName("user") var user: User? = null,
    @SerializedName("createdAt") var createdAt: String? = null
) : java.io.Serializable
